src/dmd/astbase.d

```diff
@@ -138,7 +138,7 @@ struct ASTBase
     extern (C++) __gshared const(StorageClass) STCStorageClass =
         (STC.auto_ | STC.scope_ | STC.static_ | STC.extern_ | STC.const_ | STC.final_ | STC.abstract_ | STC.synchronized_ | STC.deprecated_ | STC.override_ | STC.lazy_ | STC.alias_ | STC.out_ | STC.in_ | STC.manifest | STC.immutable_ | STC.shared_ | STC.wild | STC.nothrow_ | STC.nogc | STC.pure_ | STC.ref_ | STC.return_ | STC.tls | STC.gshared | STC.property | STC.safe | STC.trusted | STC.system | STC.disable);
 
-    enum ENUMTY : int
+    enum TY : ubyte
```

Comment:
Let's not do grand renamings.

Reply:
This is not a grand renaming, a grand renaming would be replacing `TY` with `ENUMTY`. Also this is the price to pay for not having two enums for the same thing.